app/controllers/audio_files_controller.rb

```diff
@@ -0,0 +1,22 @@
+class AudioFilesController < ApplicationController
+  def new
+    @audio_file = AudioFile.new
+  end
+
+  def create
+    @audio_file = AudioFile.new(audio_file_params)
+    @audio_file.user_id = current_user.id
```

Comment:
Consider consolidating this line and the one above by:

1. Adding a `has_many :audio_files` relation to `User.
2. Using the following:

```ruby
@audio_file = current_user.audio_files.new(audio_file_params)
```